checkpoints/checkpoint_3/js/script.js

```diff
@@ -0,0 +1,138 @@
+ 'use strict';
+ //dieChar variable holds the array of dice unicodes
+ let dieChar = ["&#9856;", "&#9857;", "&#9858;", "&#9859;", "&#9860;", "&#9861;"]
+
+ // function rolls the dice by generating a random number and places the rolled die into .die
+ function roll() {
+   document.querySelector(".dice").innerHTML = ""; // clears .dice on roll
+   document.querySelector("#results").innerHTML = ""; // clears #results on roll
+   for (let i = 0; i < 5; i++) {
+     let roll = Math.floor(Math.random() * 6) + 1; //math floor function returns the largest integer less than or equal to a given number. Math.random()*6 + 1 gives us a random number between 1 and 6.999999
+     document.querySelector(".dice").innerHTML += `<span class="die" data-roll="${roll}">${dieChar[roll - 1]}</span>`;
+   }
+   report();
+ }
+
+ roll(); //call function onclick in html
+
+
+
+ // report function reports the results of each dice roll
+ function report() {
```

Comment:
Nice. the only thing that is missing is if i hit a re-roll and get a goal again, it moves on the left side and goes to the bottom.
Nice job.